Jenkinsfile

```diff
@@ -12,8 +12,10 @@ pipeline {
     CF_HOME='/home/node' // Revert override from Jenkins global env
     CF_API="${env.CF_API}"
     CF_LOGIN=credentials('portaljs.cloudfoundry.login')
+    CF_LOG_DRAINER_SERVICE="${CF_LOG_DRAINER_SERVICE_NAME}"
```

Comment:
This should be `    CF_LOG_DRAINER_SERVICE="${env.CF_LOG_DRAINER_SERVICE_NAME}"`